Windows: tor-src/src/lib/fs/path.c:521: undefined reference to `__imp_PathMatchSpecA'

hey!

hopefully I'm opening this in the right repo, I'm trying to cross-compile a program using libtor to windows with mingw64 on Arch Linux, everything seems to build but the final linking step fails:

```
   Compiling redacted v0.2.0 (/redacted)
error: linking with `x86_64-w64-mingw32-gcc` failed: exit code: 1
  |
  = note: "x86_64-w64-mingw32-gcc" "-fno-use-linker-plugin" "-Wl,--nxcompat" "-Wl,--dynamicbase" "-Wl,--disable-auto-image-base" "-m64" "-Wl,--high-entropy-va" [... lots of text ...] "-Wl,--end-group" "/home/user/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-pc-windows-gnu/lib/libcompiler_builtins-ae4f14151f891fbb.rlib" "-Wl,-Bdynamic" "-ladvapi32" "-liphlpapi" "-lws2_32" "-lkernel32" "-lwinapi_advapi32" "-lwinapi_cfgmgr32" "-lwinapi_fwpuclnt" "-lwinapi_gdi32" "-lwinapi_kernel32" "-lwinapi_msimg32" "-lwinapi_ntdll" "-lwinapi_opengl32" "-lwinapi_user32" "-lwinapi_winspool" "-lwinapi_ws2_32" "-ladvapi32" "-lws2_32" "-luserenv" "-lgcc_eh" "-l:libpthread.a" "-lmsvcrt" "-lmingwex" "-lmingw32" "-lgcc" "-lmsvcrt" "-luser32" "-lkernel32" "/home/user/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-pc-windows-gnu/lib/rsend.o"
  = note: /usr/lib/gcc/x86_64-w64-mingw32/10.2.0/../../../../x86_64-w64-mingw32/bin/ld: /home/user/redacted/target/x86_64-pc-windows-gnu/debug/deps/libtor_sys-017306920afd59ae.rlib(path.o): in function `unglob_win32':
          /home/user/redacted/target/x86_64-pc-windows-gnu/debug/build/libtor-sys-d60e56728bfe65d1/out/tor-src/src/lib/fs/path.c:521: undefined reference to `__imp_PathMatchSpecA'
          collect2: error: ld returned 1 exit status
          

error: aborting due to previous error
```

Any ideas what I might be doing wrong?

Thanks!

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "libtor-sys"
-version = "45.7.0+0.4.5.7"
+version = "45.7.1+0.4.5.7"
 authors = ["Alekos Filini <alekos.filini@gmail.com>"]
 license = "MIT"
 homepage = "https://github.com/MagicalBitcoin/libtor-sys"

--- a/build.rs
+++ b/build.rs
@@ -300,6 +300,8 @@ fn build_tor(libevent: Artifacts) {
 
         println!("cargo:rustc-link-lib=static={}", "shell32");
         println!("cargo:rustc-link-lib=static={}", "ssp");
+
+        println!("cargo:rustc-link-lib={}", "shlwapi");
     }
 
     fs::create_dir_all(tor.join("include")).unwrap();

--- a/build.rs
+++ b/build.rs
@@ -293,13 +293,13 @@ fn build_tor(libevent: Artifacts) {
             }
         }
 
-        println!("cargo:rustc-link-lib=static={}", "crypt32");
-        println!("cargo:rustc-link-lib=static={}", "iphlpapi");
-        println!("cargo:rustc-link-lib=static={}", "ws2_32");
-        println!("cargo:rustc-link-lib=static={}", "gdi32");
+        println!("cargo:rustc-link-lib={}", "crypt32");
+        println!("cargo:rustc-link-lib={}", "iphlpapi");
+        println!("cargo:rustc-link-lib={}", "ws2_32");
+        println!("cargo:rustc-link-lib={}", "gdi32");
 
-        println!("cargo:rustc-link-lib=static={}", "shell32");
-        println!("cargo:rustc-link-lib=static={}", "ssp");
+        println!("cargo:rustc-link-lib={}", "shell32");
+        println!("cargo:rustc-link-lib={}", "ssp");
 
         println!("cargo:rustc-link-lib={}", "shlwapi");
     }

